apps/test/netsim/NetSimLogger.js

```diff
@@ -15,8 +26,8 @@ var OldConsole = function () {
 };
 
 // Modern console with log, info, warn, error methods.
-var ModernConsole = function () {
-  var that = OldConsole();
```

Comment:
:stuck_out_tongue_closed_eyes: Any action you want taken here?  I know it's clumsy, but it's just for a tiny sensing interface.